src/components/Camera/hooks/usePrepareStoreAndNavigate.ts

```diff
@@ -93,14 +95,25 @@ const usePrepareStoreAndNavigate = ( ): Function => {
         position: 0,
         local: true
       } );
+    if ( !isDefaultMode
+        && screenAfterPhotoEvidence === SCREEN_AFTER_PHOTO_EVIDENCE.OBS_EDIT
+        && visionResult ) {
+      console.log( visionResult.taxon, "ai camera suggestion" );
```

Comment:
Remove log

Reply:
Once upon a time I had a precommit hook to make sure we didn't accidentally commit console.logs...